packages/eslint-plugin/docs/rules/explicit-member-accessibility.md

```diff
@@ -310,6 +311,25 @@ class Animal {
 }
 ```
 
+### Except specific methods
+
+If you want to ignore some specific methods, you can do it by specifing method names.
```

Comment:
I'd like to clarify that this is a shotgun approach and could lead to missed lint options.

```suggestion
If you want to ignore some specific methods, you can do it by specifing method names. Note that this option does not care for the context, and will ignore every method with these names, which could lead to it missing some cases. You should use this sparingly.
```

Reply:
Done!